packages/sdk/src/Incentives/ERC20Incentive.ts

```diff
@@ -482,21 +483,24 @@ export class ERC20Incentive extends DeployableTarget<
  * @param {StrategyType} param0.strategy - The type of disbursement strategy for the incentive. `StrategyType.MINT` is not supported for `ERC20Incentives`
  * @param {bigint} param0.reward - The amount of the asset to distribute.
  * @param {bigint} param0.limit - How many times can this incentive be claimed.
+ * @param {Address} [param0.manager=zeroAddress] - The entity that can draw raffles - defaults to unset
  * @returns {Hex}
  */
 export function prepareERC20IncentivePayload({
   asset,
   strategy,
   reward,
   limit,
+  manager = zeroAddress,
```

Comment:
@sammccord I set this to a sensible default so if someone _doesn't_ set this raffles won't work. Mostly so for the non-raffle incentives we don't have to futz with this.

Reply:
Then you'll also want to make the `manager` field in the `ERC20IncentivePayload` interface on line 85 optional as well